docker/wc-admin-docker-dev/README.md

```diff
@@ -0,0 +1,111 @@
+# WooCommerce Docker Setup with WP-ENV
+
+Docker development setup for WooCommerce Admin with WP-ENV.
+
+## Prerequisites
+
+Please install WP-ENV before getting started. You can find more about WP-ENV on [here](https://github.com/WordPress/gutenberg/tree/master/packages/env).
+
+The following command installs WP-ENV globally.
+
+`npm -g i @wordpress/env`
+
+
+## Getting Started
+
+1. Navigate to the root of WooCommerce Admin source code.
+2. Start the docker container by running `wp-env start`
+
+You should see the following output
+
+```
+WordPress development site started at http://localhost:8888/
+WordPress test site started at http://localhost:8889/
+MySQL is listening on port 55003
+```
+
+The port # might be different depending on your `.wp-env.override.json` configuration.
+
+## Overriding the Default Configuration
+
+The default configuration comes with PHP 7.4, WooCommerce 5.0, and a few WordPress config values.
+
+You can create `.wp-env.override.json` file and override the default configuration values.
+
+You can find more about `.wp-env.override.json` configuration [here](https://github.com/WordPress/gutenberg/tree/master/packages/env#wp-envoverridejson).
```

Comment:
Its great that we link to the docs, but I wonder if a couple of things from the docs deserve highlighting here to help newbies out. Two that come to mind:

1. Adding other local plugins to your override so they can be devved on to. Great if you have a local copy of woocommerce and woocommerce-admin that you want to dev on together

2. Setting up xdebug. This is something that can be a bit daunting but its really important to have access to a working xdebug config. It could just be as simple as explicitly linking this section: https://github.com/WordPress/gutenberg/tree/master/packages/env#using-xdebug

Reply:
Good idea 👍 I will update the doc.